Disable this.

This works but wants to open LICENSE.html. Which also works in theory but
LO hangs for confirmation whether we really should open this read-only (of course it's read-only,
the file is in /usr!) and the test "hangs".

diff --git a/sc/qa/uitest/calc_dialogs/openDialogs.py b/sc/qa/uitest/calc_dialogs/openDialogs.py
index f616c67f7c6c..24d828953f48 100644
--- a/sc/qa/uitest/calc_dialogs/openDialogs.py
+++ b/sc/qa/uitest/calc_dialogs/openDialogs.py
@@ -157,7 +157,6 @@ dialogs = [
         # tested in sc/qa/uitest/protect/protectSpreadsheet.py
     {"command": ".uno:RunMacro", "closeButton": "cancel"},
     {"command": ".uno:ScriptOrganizer", "closeButton": "close"},
-    {"command": ".uno:ShowLicense", "closeButton": "close"},
     # {"command": ".uno:About", "closeButton": "close"},
         # tested in sc/qa/uitest/calc_tests7/tdf106667.py
 ]
diff --git a/sw/qa/uitest/writer_dialogs/openDialogs.py b/sw/qa/uitest/writer_dialogs/openDialogs.py
index b4d9b356e153..2523b7b9bf98 100644
--- a/sw/qa/uitest/writer_dialogs/openDialogs.py
+++ b/sw/qa/uitest/writer_dialogs/openDialogs.py
@@ -113,7 +113,6 @@ dialogs = [
         # tested in uitest/writer_tests/customizeDialog.py
     #{"command": ".uno:OptionsTreeDialog", "closeButton": "cancel"},
         # used in various places
-    {"command": ".uno:ShowLicense", "closeButton": "close"},
     # {"command": ".uno:About", "closeButton": "close"},
         # tested in sw/qa/uitest/writer_tests5/about_test.py
 ]
